dev-libs/pigpio/pigpio-67.ebuild

```diff
@@ -21,7 +21,7 @@ DEPEND=""
 RDEPEND="${DEPEND}"
 
 src_prepare() {
-	eapply "${FILESDIR}/${P}-makefile.patch"
+	eapply "${FILESDIR}/${PN}-makefile.patch"
```

Comment:
Instead of renaming the patch, just replace `${P}` with the specific version in the new ebuild.